public/api-docs/index.yaml

```diff
@@ -7,13 +7,188 @@ info:
     name: MIT
     url: "https://opensource.org/license/mit/"
 servers:
-  - url: http://localhost:8080
+  - url: http://localhost:8080/api/v1
     description: local server
 
-# paths:
-#   /:
-#     # define route paths
+paths:
+  /Languages:
+    get:
+      summary: "Retrieve a list of languages"
+      description: "Returns a list of language codes and labels available in the system."
+      responses:
+        '200':
+          description: "A list of languages"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Language'
+              examples:
+                english:
+                  summary: "Example - English"
+                  value:
+                    - languageCode: "en"
+                      languageLabel: "English"
+                spanish:
+                  summary: "Example - Spanish"
+                  value:
+                    - languageCode: "es"
+                      languageLabel: "Spanish"
+                french:
+                  summary: "Example - French"
+                  value:
+                    - languageCode: "fr"
+                      languageLabel: "French"
+                german:
+                  summary: "Example - German"
+                  value:
+                    - languageCode: "de"
+                      languageLabel: "German"
+        '400':
+          description: "Invalid input"
+        '500':
+          description: "Server error"
 
-# components:
-#   schemas:
-#     # Define any reusable components or schemas
+  /match/list:
+    get:
+      summary: "Retrieve a list of matched pronunciation audios"
+      description: "Returns a list of matches between forms and audio files."
+      responses:
+        '200':
+          description: "A list of matches"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Match'
+        '400':
+          description: "Invalid input"
+        '500':
+          description: "Server error"
+
+  /match/publish:
+    post:
+      summary: "Publish a match between a form and an audio file"
+      description: "Creates a new match between a form and an audio file, along with the user who published it."
+      requestBody:
+        required: true
+        content:
```

Comment:
We need to tell frontend guys what the body should look like.

Reply:
working on it sir